notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>accompanying text cell should explain parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>